presto-main/src/main/java/io/prestosql/execution/QueryManagerConfig.java

```diff
@@ -295,6 +297,18 @@ public QueryManagerConfig setQueryMaxCpuTime(Duration queryMaxCpuTime)
         return this;
     }
 
+    public DataSize getQueryMaxScanPhysicalBytes()
+    {
+        return queryMaxScanPhysicalBytes;
+    }
+
+    @Config("query.max-scan-physical-bytes")
+    public QueryManagerConfig setQueryMaxScanPhysicalBytes(DataSize queryMaxScanPhysicalBytes)
+    {
+        this.queryMaxScanPhysicalBytes = queryMaxScanPhysicalBytes;
```

Comment:
`this.queryMaxScanPhysicalBytes = Optional.ofNullable(queryMaxScanPhysicalBytes)`

Reply:
Done